src/wallet/CMakeLists.txt

```
@@ -0,0 +1,59 @@
+# Copyright (c) 2023-present The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://opensource.org/license/mit/.
+
+# Wallet functionality used by bitcoind and bitcoin-wallet executables.
+add_library(bitcoin_wallet STATIC EXCLUDE_FROM_ALL
+  coincontrol.cpp
+  coinselection.cpp
+  context.cpp
+  crypter.cpp
+  db.cpp
+  dump.cpp
+  external_signer_scriptpubkeyman.cpp
+  feebumper.cpp
+  fees.cpp
+  interfaces.cpp
+  load.cpp
+  migrate.cpp
+  receive.cpp
+  rpc/addresses.cpp
+  rpc/backup.cpp
+  rpc/coins.cpp
+  rpc/encrypt.cpp
+  rpc/spend.cpp
+  rpc/signmessage.cpp
```

Nit, feel free to ignore these if not important:
```suggestion
  rpc/spend.cpp
  rpc/signmessage.cpp
```

---

See https://github.com/hebasto/bitcoin/pull/320.